rustdoc-json/src/lib.rs

```diff
@@ -38,6 +38,10 @@ pub enum BuildError {
     #[error("Failed to build rustdoc JSON: {0}")]
     General(String),
 
+    /// An error originating from building the crate's documentation.
```

Comment:
Let's move my comment here:

```suggestion
    /// An error originating from building the crate's documentation.
    /// In this case the exact error can be seen on stderr.
```

Reply:
Done